### DISCUSSION TITLE
[Enhancement] [check-tag] 希望el-check-tag增加一个type属性控制背景色

### DISCUSSION THREAD

**Discussion by Greenjiao** [element-plus/element-plus#15709] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `check-tag`

## Description

check-tag使用时有时会需要根据不同类型设置tag选中后的背景色，现在没看见有提供类似普通tag的type属性来控制，希望能增加一下
```vue
<template>
  <el-check-tag checked type="danger">Checked</el-check-tag>
</template>
```

### Additional comments

_(empty)_

**Comment by kooriookami** [comment]
#15727
